src/test/resources/application-test.yml

```diff
@@ -1,16 +1,12 @@
 spring:
-
-  config:
-    use-legacy-processing: true
-
   security:
     oauth2:
       resourceserver:
         jwt:
           public-key-location: classpath:local-public-key.pub
 
   datasource:
-    url: 'jdbc:h2:mem:licences-db;MODE=PostgreSQL'
+    url: 'jdbc:h2:mem:licences-db;MODE=PostgreSQL;DB_CLOSE_DELAY=-1'
```

Comment:
Had to add this as otherwise it would create a new instance of H2 without the schema which caused integration tests to fail as it was querying an empty DB